### PR TITLE
RQ-1182: Homepage help changelog cards

### PR DESCRIPTION
**Pull request by RuntimeTerror10**
Closes issue: 

## ✅ Checklist:

- [ ] Make sure linting and unit tests pass.
- [ ] No install/build warnings introduced.
- [ ] Verified UI in browser.
- [ ] For UI changes, added/updated analytics events (if applicable).
- [ ] For changes in extension's code, manually tested in Chrome and Firefox.
- [ ] For changes in extension's code, both MV2 and MV3 are covered.
- [ ] Added/updated unit tests for this change.
- [ ] Raised pull request to update corresponding documentation (if already exists).

### CODE REVIEW

**Comment by rohanmathur91** [comment]
Box-shadows are missing for all the cards

File: app/src/components/Home/assets/book.svg
Comment (rohanmathur91) [inline app/src/components/Home/assets/book.svg:1]: Why are we not using react-icons?

File: app/src/components/Home/components/ChangelogCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import { ChangeLogType, Changelog } from "components/Home/types";
+import Logger from "lib/logger";
+import "./changelogCard.scss";
+
+export const ChangeLogCard: React.FC = () => {
+  const [logs, setLogs] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetch("https://requestly.tech/api/mockv2/v1/rq-changelog?teamId=LmyapmzY39XTFXvua6eX")
```
Comment (rohanmathur91) [inline app/src/components/Home/components/ChangelogCard/index.tsx:13]: How we are going to maintain the changelog?

File: app/src/components/Home/components/HelpCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import { Link } from "react-router-dom";
+import { Col } from "antd";
+import callIcon from "../../assets/call.svg";
+import infoIcon from "../../assets/info.svg";
+import communityIcon from "../../assets/community.svg";
+import documentIcon from "../../assets/document.svg";
+import bookIcon from "../../assets/book.svg";
+import "./helpCard.scss";
+
+const helperLinks = [
```
Comment (rohanmathur91) [inline app/src/components/Home/components/HelpCard/index.tsx:11]: keep this inside the component, in useMemo.

**Review by rohanmathur91 — APPROVED** [review]

File: app/src/components/Home/components/ChangelogCard/index.tsx (hunk 1 above)
Comment (RuntimeTerror10) [reply app/src/components/Home/components/ChangelogCard/index.tsx:13]: I have created a mock API in RQ official workspace and added a static JSON response. We just need to update the fields.

File: app/src/components/Home/assets/book.svg
Comment (RuntimeTerror10) [reply app/src/components/Home/assets/book.svg:1]: Some of the icons are not available in react-icons, so different styling is required to handle to react-icons and downloaded svg codes.

**Pull request merged**